apps/hash-agents/app/__init__.py

```diff
@@ -37,7 +37,7 @@ def agent(agent_name):
         try:
             return call_agent(agent_name, **request.json)
         except Exception as e:
-            getLogger(__name__).error(e)
+            getLogger(__name__).critical(e, exc_info=True)
```

Comment:
I would probably keep it as an `error` as this is not critical for the service itself, the service will still work.
```suggestion
            getLogger(__name__).error(e, exc_info=True)
```